catalog/admin/admin-poller-service/src/test/java/org/codice/ddf/catalog/admin/poller/AdminPollerServiceBeanTest.java

```diff
@@ -145,10 +160,57 @@ public void testAllSourceInfo() {
   }
 
   @Test
-  public void testSourceStatus() {
+  public void testSourceStatusAvailable() throws Exception {
+    SourceDescriptorImpl Descriptor = mock(SourceDescriptorImpl.class);
```

Comment:
❓ Why do we mock the `SourceDescriptorImpl` here and not the `SourceDescriptor`? Should mock the interface if we can.